pdu/ShortMessage.go

```diff
@@ -84,6 +111,25 @@ func (c *ShortMessage) SetMessageData(data []byte) {
 	c.messageData = data
 }
 
+// GetMessageData returns underlying binary message.
+func (c *ShortMessage) GetMessageData() (data []byte, err error) {
+	if len(c.messageData) == 0 {
+		return
```

Comment:
maybe return empty byte slice instead of nil is more meaningful,

If error occurs then ``` data == nil && err != nil```

Reply:
The default value of a []byte is empty slice, not nil ;)

Reply:
hmmm i was testing this before https://play.golang.org/p/AHVzI3Gjjre

Reply:
My bad ... I fix it